test/basic.html

```diff
@@ -1,30 +1,148 @@
-				});
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+	<title>cosmoz-omnitable-treenode-column basic tests</title>
+
+	<script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+	<script src="../../web-component-tester/browser.js"></script>
+	<script src="../../test-fixture/test-fixture-mocha.js"></script>
+
+	<link rel="import" href="../../test-fixture/test-fixture.html">
+	<link rel="import" href="../../iron-test-helpers/iron-test-helpers.html">
+
+	<link rel="import" href="../../cosmoz-omnitable/cosmoz-omnitable.html">
+	<link rel="import" href="../../cosmoz-omnitable/cosmoz-omnitable-templatizer.html">
+	<link rel="import" href="../../iron-ajax/iron-ajax.html">
+	<link rel="import" href="../../cosmoz-tree/cosmoz-default-tree.html">
+	<link rel="import" href="../cosmoz-omnitable-treenode-column.html">
+</head><body>
+
+	<test-fixture id="basicTree">
+		<template>
+			<iron-ajax auto
+				url="../../cosmoz-tree/demo/tree.json"
+				handle-as="json">
+			</iron-ajax>
+		</template>
+	</test-fixture>
+
+	<test-fixture id="basic">
+		<template>
+			<cosmoz-omnitable class="fit" style="height:300px; width:1200px;">
+				<cosmoz-omnitable-column name="name" title="Name" value-path="name">
+				</cosmoz-omnitable-column>
+				<cosmoz-omnitable-treenode-column name="node" title="Node" flex="5" show-max-nodes="3"
+					value-path="nodeId" key-property="id" show-results-on-focus>
+				</cosmoz-omnitable-treenode-column>
+			</cosmoz-omnitable>
+		</template>
+	</test-fixture>
+
+	<script>
+	suite('basic', function () {
+		var omnitable,
+			column,
+			tree,
+			paperAutocomplete,
+			data = [
+				{
+					name: 'Sed a quam interdum',
+					nodeId: '167d1485-7d4f-4c7d-86cd-a4fb00f31245'
+				},
+				{
+					name: 'Euismod sapien non',
+					nodeId: '3a7654f1-e3e6-49c7-b6a8-a4fb00f31245'
+				},
+				{
+					name: 'rhoncus dolor',
+					nodeId: '426d7dba-a4f0-4bd4-9b34-a4fb00f31248'
+				},
+				{
+					name: 'Duis mollis leo ut nibh sagittis',
+					nodeId: '1641faed-1043-450e-868d-a4fb00f3124f'
+				},
+				{
+					name: 'a porta leo maximus',
+					nodeId: '53f0b29d-6755-4099-b70c-a4fb00f3124e'
+				},
+				{
+					name: 'Fusce tristique leo',
+					nodeId: 'a0ae20d8-7074-4bfb-9864-a4fb00f3124d'
+				},
+				{
+					name: 'a porttitor porttitor',
+					nodeId: '81e3a3ef-abae-4781-aa4c-a4fb00f31252'
+				},
+				{
+					name: 'Nam finibus leo euismod',
+					nodeId: 'abbe92c9-d7f4-448e-a531-a4fb00f31249'
+				},
+				{
+					name: 'Sed luctus leo sollicitudin',
+					nodeId: '8d55d691-b8a8-4292-ad4a-a4fb00f3124e'
+				},
+				{
+					name: 'Cras laoreet erat ut eros viverra egestas',
+					nodeId: '6fd57f4f-43e9-4a4c-800c-a4fb00f31246'
+				},
+				{
+					name: 'Etiam sit amet mauris semper',
+					nodeId: '219b3399-7edd-42b7-ad92-a4fb00f31247'
+				},
+			];
+
+		this.timeout(9e2000);
```

Comment:
Was this for testing purpose ? Or does the test takes really a long time to run ?